### DISCUSSION TITLE
Mass Upload of Registrants?

### DISCUSSION THREAD

**Discussion by mad2492** [SFDO-Community-Sprints/Summit-Events-App#500] [General]
We wanted to import registrants from future events from our old CRM into the new Summit Event instances. 
Is there any recommendations or things to be wary of when doing this?

**Comment by dwhitig** [comment]
@tcdahlberg @rdblake21 Any thoughts here to add?  @mad2492 what is your events structure?  Are they mostly single events or multi-day events?  What type of events are they (e.g. webinars, campus visits, in-person events, etc.)?

**Reply by rdblake21** [reply]
@walt0019 may have some good advice here knowing he's done this in past (and maybe recently)

**Reply by walt0019** [reply]
Hi @mad2492,

We have done this process a lot.

I used dataloader to load the registrations.  Please make sure you load the Summit Event Event and Summit Event Instance record reference values on to the Registration record, otherwise you will receive errors when trying to update the status or other fields on the Registration record in the future.

I would confirm that you do not have any active Email templates set to fire on the creation of the Registration records.

When loading in the Registration records, you can still do Contact and Lead matching, but I would confirm your batch size is low if you want that to happen.

We have also mapped Qualtrics survey results into Summit Event Registrations and had a lot of success working with it, which can match your use case as well.

Please let me know if you have other questions or want to set up a quick meeting at some point.  I'm going to try to burn some vacation days before the end of our fiscal year, and don't plan to be in after today this week, but will be in after the Fourth.

**Comment by mad2492** [comment]
> Hi @mad2492,
> 
> We have done this process a lot.
> 
> I used dataloader to load the registrations. Please make sure you load the Summit Event Event and Summit Event Instance record reference values on to the Registration record, otherwise you will receive errors when trying to update the status or other fields on the Registration record in the future.
> 
> I would confirm that you do not have any active Email templates set to fire on the creation of the Registration records.
> 
> When loading in the Registration records, you can still do Contact and Lead matching, but I would confirm your batch size is low if you want that to happen.
> 
> We have also mapped Qualtrics survey results into Summit Event Registrations and had a lot of success working with it, which can match your use case as well.
> 
> Please let me know if you have other questions or want to set up a quick meeting at some point. I'm going to try to burn some vacation days before the end of our fiscal year, and don't plan to be in after today this week, but will be in after the Fourth.

Hi! Thank you so much! This is definitely what I was looking for. I could have done a lot of trial and error on my side, but for my own sanity I was hoping this community had some tips & tricks to save me some time. I'll report back with my results or problems haha
